### ISSUE TITLE
warnings for missing required properties need to be clearer separated from real errors

### ISSUE THREAD

**Issue by lzehl** [openMetadataInitiative/openMINDS_Python#32]
The issue / warning that is given for a missing required property needs to be clearer formulated as a warning instead of looking as an error.

**Comment by apdavison** [comment]
I don't think this issue is relevant anymore, since missing required properties are handled by the `validate()` method now, the same as all other validation failures. 

Feel free to re-open if I've misunderstood.
